cmd/rest/main.go

```diff
@@ -20,6 +20,11 @@ func main() {
 // Run will bind our layers all together
 func Run() error {
 	// TODO: Handle errors, migration, configs ...
+	err := cfg.LoadEnvVar()
```

Comment:
Can we do some kind of encapsulation here?
I want to have the only function - GetConfig.
If you need to load env variables - pls do that inside that function.

Reply:
I hope I got the point right.